REFORGED/StandardUnits.txt

```diff
@@ -1,425 +1,425 @@
+UPG_BLK_SPHINX	Rusp	UNDEAD	upgrade	UPGRADE	SLAUGHTERHOUSE	NECROPOLIS_3	TOMB_OF_RELICS	0	0	false	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
+UPG_BURROWING	Rubu	UNDEAD	upgrade	UPGRADE	CRYPT	NECROPOLIS_2	GRAVEYARD	0	0	false	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
+UPG_EXHUME	Ruex	UNDEAD	upgrade	UPGRADE	SLAUGHTERHOUSE	NECROPOLIS_2	0	0	0	false	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
+UPG_UNHOLY_STR	Rume	UNDEAD	upgrade	ML_UPGRADE	GRAVEYARD	NECROPOLIS_2	NECROPOLIS_3	0	0	false	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
+UPG_CR_ATTACK	Rura	UNDEAD	upgrade	ML_UPGRADE	GRAVEYARD	NECROPOLIS_2	NECROPOLIS_3	0	0	false	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
+UPG_UNHOLY_ARMOR	Ruar	UNDEAD	upgrade	ML_UPGRADE	GRAVEYARD	NECROPOLIS_2	NECROPOLIS_3	0	0	false	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
+UPG_CR_ARMOR	Rucr	UNDEAD	upgrade	ML_UPGRADE	GRAVEYARD	NECROPOLIS_2	NECROPOLIS_3	0	0	false	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
+UPG_NECROS	Rune	UNDEAD	upgrade	ML_UPGRADE	DAMNED_TEMPLE	NECROPOLIS_3	0	0	0	false	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
+UPG_BANSHEE	Ruba	UNDEAD	upgrade	ML_UPGRADE	DAMNED_TEMPLE	NECROPOLIS_3	0	0	0	false	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
+DEMON_HUNTER	Edem	ELF	hero	HERO	ELF_ALTAR	TREE_AGES	TREE_ETERNITY	0	0	false	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
+DEMON_HUNTER_M	Edmm	ELF	hero,mutated	HERO	ELF_ALTAR	TREE_AGES	TREE_ETERNITY	0	0	false	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
+KEEPER	Ekee	ELF	hero	HERO	ELF_ALTAR	TREE_AGES	TREE_ETERNITY	0	0	false	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
+MOON_PRIESTESS	Emoo	ELF	hero	HERO	ELF_ALTAR	TREE_AGES	TREE_ETERNITY	0	0	false	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
+WARDEN	Ewar	ELF	hero	HERO	ELF_ALTAR	TREE_AGES	TREE_ETERNITY	0	0	false	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
+WISP	ewsp	ELF	peon	UNIT	TREE_LIFE	0	0	0	0	false	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0
+ARCHER	earc	ELF	0	UNIT	ANCIENT_WAR	0	0	0	0	true	0	0	0	0	0	0	0	0	2	0	0	0	0	0	2	0
+DRUID_TALON	edot	ELF	0	UNIT	ANCIENT_WIND	0	0	0	DRUID_TALON_M	true	0	0	0	0	0	0	2	0	0	0	0	2	2	0	0	0
+DRUID_TALON_M	edtm	ELF	mutated	UNIT	0	0	0	0	DRUID_TALON	true	0	0	0	0	0	2	2	0	2	0	0	0	2	0	0	0
+BALLISTA	ebal	ELF	siege	UNIT	ANCIENT_WAR	HUNTERS_HALL	0	0	0	true	0	0	0	0	0	0	0	0	0	0	3	0	0	0	0	3
+DRUID_CLAW	edoc	ELF	0	UNIT	ANCIENT_LORE	0	0	0	DRUID_CLAW_M	true	0	0	0	0	0	0	4	0	0	4	0	0	0	0	0	4
+DRUID_CLAW_M	edcm	ELF	mutated	UNIT	0	0	0	0	DRUID_CLAW	true	0	0	0	0	0	0	4	0	0	4	0	0	0	0	0	4
+DRYAD	edry	ELF	0	UNIT	ANCIENT_LORE	0	0	0	0	true	0	0	0	0	0	0	0	0	3	0	0	0	3	0	0	0
+HIPPO	ehip	ELF	0	UNIT	ANCIENT_WIND	0	0	0	0	true	0	0	0	0	0	2	0	0	0	2	0	0	2	0	0	0
+HIPPO_RIDER	ehpr	ELF	0	UNIT	HIPPO	ARCHER	0	0	0	true	0	0	0	0	0	4	0	0	4	0	0	0	0	4	0	0
+HUNTRESS	esen	ELF	0	UNIT	ANCIENT_WAR	HUNTERS_HALL	0	0	0	true	0	0	0	0	0	0	0	0	0	3	0	0	3	0	0	0
+CHIMAERA	echm	ELF	siege	UNIT	CHIMAERA_ROOST	0	0	0	0	true	0	0	0	0	0	5	0	0	0	0	0	5	0	5	0	0
+ENT	efon	ELF	0	UNIT	KEEPER	0	0	0	0	false	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	1
+MOUNTAIN_GIANT	emtg	ELF	0	UNIT	ANCIENT_LORE	TREE_AGES	DEN_OF_WONDERS	0	0	true	0	0	0	0	0	0	0	0	0	7	3	0	0	0	7	0

```

Comment:
Unessecary, ancient of lore already requires tree of ages

Reply:
It is necessary to avoid the situation where the main base is demolished but the barracks still exist, and training is no longer possible at this time